src/bridge.cpp

```diff
@@ -55,13 +77,42 @@ create_bridge_from_2_to_1(
   const std::string & ros1_topic_name,
   size_t publisher_queue_size,
   rclcpp::PublisherBase::SharedPtr ros2_pub)
+{
+  rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_sensor_data;
+  custom_qos_profile.depth = subscriber_queue_size;
+  auto subscriber_qos = rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(custom_qos_profile));
+  subscriber_qos.get_rmw_qos_profile() = custom_qos_profile;
```

Comment:
```suggestion
  auto subscriber_qos = rclcpp::SensorDataQoS(subscriber_queue_size);
```

Reply:
Thanks - applied in 5b2dcabf6827dd8f27feae567ea5b7136db9e421.

Reply:
It seems that `SensorDataQoS` doesn't have a constructor taking a queue size. I modified the line in 6c256577384324162d3f149d0e920d452c694a00.

I also applied the same change to existing code in the `factory.hpp`: e3ca885d990382f9614713ac2a2e873bc3ae4262.